ops/testing.py

```diff
@@ -1238,8 +1242,9 @@ def storage_add(self, name: str, count: int = 1):
         for i in range(count):
             index = self._storage_id_counter
             self._storage_id_counter += 1
+            storage_dir = os.path.join(self._harness_tmp_dir.name, name, str(index))
```

Comment:
Should we protect here against `cd`-ing out of `/tmp`?
```python
>>> os.path.join('/tmp/temp', '//weird//name')
'//weird//name'
```

Reply:
Perhaps we could add an easter egg test writers: `error: why you puttin' slashes in test container names?`

Reply:
I added a check that errors for absolute paths